### ISSUE TITLE
How to remove fixed pattern noise?

### ISSUE THREAD

**Issue by emilfihlman** [melexis/mlx90640-library#119]
How to reduce / remove it? Am I doing my data reading correctly?
[Screencast from 2024-10-22 18-43-48.webm](https://github.com/user-attachments/assets/989d1344-4333-4684-a1aa-377e8f56e970)

**Comment by slavysis** [comment]
Hi, did you put a decoupling capacitor between Vdd and GND? Normally 100nF+10uF as close as possible to the pins should be enough.

Best regards

**Comment by emilfihlman** [comment]
@slavysis
Yeah I have 10uF, 100n and 47n right at the base of the device, still same result.
[Screencast from 2024-10-23 13-59-37.webm](https://github.com/user-attachments/assets/e5a478e8-1751-48b8-8f02-1b0c35d986d8)
